src/vs/platform/clipboard/browser/clipboardService.ts

```diff
@@ -28,31 +34,64 @@ export class BrowserClipboardService implements IClipboardService {
 		try {
 			return await navigator.clipboard.writeText(text);
 		} catch (error) {
-			console.error(error);
-		}
-
-		// Fallback to textarea and execCommand solution
-
-		const activeElement = document.activeElement;
-
-		const textArea: HTMLTextAreaElement = document.body.appendChild($('textarea', { 'aria-hidden': true }));
-		textArea.style.height = '1px';
-		textArea.style.width = '1px';
-		textArea.style.position = 'absolute';
+			if (!isSafari) {
```

Comment:
I wonder if we should always show the dialog? There are chances when desktop browsers (e.g., Firefox) think they should block the clipboard access and we may always want to offer users a workaround.

Reply:
:thumbsup: